src/Esfa.Vacancy.Api.Types/ApplicationMethod.cs

```diff
@@ -0,0 +1,9 @@
+﻿namespace Esfa.Vacancy.Api.Types
+{
+    public enum ApplicationMethod
+    {
+        NotSpecified = 0,
+        Online = 1,
+        Offline = 2
```

Comment:
Since we haven't done OffLine, Just OnLine  should be in the enum. Not specified is not required to be in the enum either.

Reply:
Done